dbms/src/Storages/DeltaMerge/StableValueSpace.h

```diff
@@ -18,7 +18,9 @@
 #include <Storages/DeltaMerge/File/ColumnCache.h>
 #include <Storages/DeltaMerge/File/DMFilePackFilter_fwd.h>
 #include <Storages/DeltaMerge/File/DMFile_fwd.h>
+#include <Storages/DeltaMerge/File/dtpb/dmfile.pb.h>
```

Comment:
No need for this include. Then we can reduce files to recompile when dmfile.pb is changed.